src/service/arguments/build/branch-command.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from "commander";
+import { CommandConstructor } from "@bc/service/arguments/command-constructor";
+import { CLIActionType, BuildActionType } from "@bc/domain/cli";
+
+/**
+ * Create branch flow sub-subcommand for build subcommand
+ * @implements {CommandConstructor}
+ */
+export class BranchCommand implements CommandConstructor {
+    createCommand(): Command {
+        const program = new Command(`${CLIActionType.BUILD} ${BuildActionType.BRANCH}`);
+        program
+            .description("Execute branch build chain workflow")
+            .requiredOption("-p, --startProject <project>", "The project to start the build from")
+            .requiredOption("-b, --branch <branch>", "The branch to get the project from")
+            .option("--fullProjectDependencyTree", "Checks out and execute the whole tree instead of the upstream build", false)
+            .option("-m, --command <commands...>", "The command(s) to execute for every project. This will override definition file configuration (just dependency tree will be taken into account)")
```

Comment:
is `m` on purpose? :thinking: 
```suggestion
            .option("-c, --command <commands...>", "The command(s) to execute for every project. This will override definition file configuration (just dependency tree will be taken into account)")
```

Reply:
I would suggest to keep `customCommandTreatment` as it actually is `cct` wdyt?

Reply:
I was trying to avoid multi character as single dash options since single dash options are generally supposed to be single character. Keeping it single character allows us to declare multiple flags in one go like for eg `ls -la` which is equivalent to `ls -l -a`.

Reply:
Yep sounds good